bin/build_docker_images.sh

```diff
@@ -1,14 +1,9 @@
 #!/usr/bin/env bash
 
-set -e
+# Show script in output, and error if anything fails
+set -xe
 
-# ensure REPO_URL is set externally (e.g. via CI/CD)
-if [ "x$REPO_URL" = "x" ]; then
-  echo "error: REPO_URL is required to be set"
-  exit 1
-fi
-
-IMAGE_NAME="dashevo/drive"
+IMAGE_NAME="dashpay/drive"
 DRIVE_VERSION=$(node -p "require('./package.json').version")
 
```

Comment:
Lets check package version and current build as in js-dpp

Reply:
That repo is setup to build on tags, but this one builds every merge to `master`. I tried to keep the logic in each repo and not change it.

That also seems like a bigger change that does more than just the PR description, which is to deploy to Docker Hub (instead of ECR). Don't you think we should keep PRs focused per our [DashCore contributing guidelines](https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#pull-request-philosophy)?

Reply:
Yeah, agree!